### DISCUSSION TITLE
Connecting to specific TurboWarp project (Cloud Variables)

### DISCUSSION THREAD

**Discussion by Mr-Programmer-777** [TurboWarp/packager#301] [Q&A] [answered]
How can I get a HTML packaged project to connect to a specific already-existing TurboWarp server? E.g I packaged a laggy Scratch/TurboWarp project and it won't connect to the server?

**Comment by GarboMuffin** [comment, marked as answer]
You're going to have to be more specific than that. You could try opening the developer console (Right click on the background around the project > Inspect element > Console tab) and see what messages appear in there (post them here)

**Reply by Mr-Programmer-777** [reply]
![Screen Shot 2022-02-13 at 02 00 04](https://user-images.githubusercontent.com/77010833/153735085-c9ec93a8-710f-4f34-8c74-ad89e7f4d548.png)

**Reply by Mr-Programmer-777** [reply]
I think I know what it is but i'm not sure

**Reply by GarboMuffin** [reply]
The server doesn't like the username you're using. Try changing it to something else.

**Reply by Mr-Programmer-777** [reply]
Left the username default
![Screen Shot 2022-02-13 at 02 12 31](https://user-images.githubusercontent.com/77010833/153735328-e5683067-e873-4b30-9eea-5bb76bfa2dc2.png)
:

**Reply by GarboMuffin** [reply]
The connection was successfully opened. Is it not working? What do you have project ID set to in advanced settings?

**Reply by GarboMuffin** [reply]
@hello-smile6 WebGL errors are completely unrelated to cloud variables

**Reply by Mr-Programmer-777** [reply]
The ID is the TurboWarp cloud I want to connect to, `469164737`

**Reply by GarboMuffin** [reply]
I just tested the project with a tab on turbowarp.org and a tab on the packager and multiplayer worked fine.

**Reply by Mr-Programmer-777** [reply]
So what am I supposed to do then

**Reply by GarboMuffin** [reply]
I pushed an update to https://packager.turbowarp.org/ with increased logging, please refresh the page, repackage the project, and take another screenshot of the logs

**Reply by Mr-Programmer-777** [reply]
Thank you so much! It works!
